packages/storybook/cxl-ui/cxl-paywall/cxl-paywall-layout=2c-l.stories.js

```diff
@@ -128,7 +128,9 @@ const Template = ({ _count, _limit, delay, opacity, subscribed }) => html`
         opacity=${opacity}
         ?subscribed=${subscribed}
       >
-        <footer class="entry-footer">${CXLPlaybookAccordion()}</footer>
+        <footer class="entry-footer">
+          ${CXLPlaybookAccordion({ FeedbackButtonLabel: 'Leave feedback', PlaybookId: '1234' })}
+        </footer>
```

Comment:
Related to this PR, or?

Reply:
The Chromatic build will fail because the missing arguments trigger a console error.